src/app/modules/products/products.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { ProductsRoutingModule } from './products-routing.module';
+import { UahCurrencyPipe } from '../../pipes/uah-currency.pipe';
+import { PriceColorDirective } from '../../directives/price-color.directive';
+
+@NgModule({
+  declarations: [UahCurrencyPipe, PriceColorDirective],
```

Comment:
In the future it will be nice to get a shared.module for UahCurrencyPipe, PriceColorDirective. So far, everything is fine.